pkg/virt-launcher/virtwrap/api/defaults.go

```diff
@@ -14,17 +14,6 @@ func SetDefaults_Devices(devices *Devices) {
 			},
 		},
 	}
-
-	// For now connect every virtual machine to the default network
-	devices.Interfaces = []Interface{{
-		Model: &Model{
-			Type: "e1000",
-		},
-		Type: "network",
-		Source: InterfaceSource{
```

Comment:
I would also prefer to add the default network interface via the defaulter and just enrich it with the mac address later on ...

Reply:
We can, it just looks weird. Today it's just a bridge, tomorrow we will have device plugins and other types of networks, why not let the networking code decide what should be configured?

Also, from my POV, we should default certain values, such as model, but not whole device configuration.

Reply:
For the pod network we can't just randomly choose on how the interface looks like. It is not yet clear to me, if and how we make it configurable on the API level. What do you think?

Reply:
At the moment we can't, but going forward, we might. Here we are laying down how the configuration will look like in the future. My fear is that later we will need to always keep in mind that we have a device which is being configured prior to the actual network configuration and it will introduce confusion.

Reply:
I don't know I don't think that this lays down anything for the future. We just have a place where all defaults are added on the host side and I think it is a good place to have it there together for as long as we do it this way, just to find all places where we modify the vm easier.